packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -355,9 +367,10 @@ export class VaFileInput {
       headless,
       value,
       readOnly,
+      statusText
     } = this;
 
-    if (value) {
+    if (value && !this.file) {
```

Comment:
This fixes an existing bug, where if the value is set on the component the user could not change the value. If the user tried to change the value, it would just revert to the original value.